src/app/dataset-block/metadata-block/metadata-block.component.ts

```diff
@@ -40,10 +47,14 @@ export class MetadataBlockComponent
     public datasetInfo: DatasetInfo;
     public datasetViewType = DatasetViewTypeEnum.History;
     public blockHash: string;
+    public datasetHistoryUpdate: DatasetHistoryUpdate;
```

Comment:
This should be a nullable type with `null` by default. Right now it hanges undefined until history query returns.

Reply:
Fixed